catalog/package.json

```diff
@@ -210,6 +211,7 @@
     "jest": "^27.5.1",
     "jest-localstorage-mock": "^2.4.21",
     "lint-staged": "^12.4.1",
+    "markdownlint-cli": "^0.32.1",
```

Comment:
Shouldn't we put it to the different place, since it lints docs for the whole project?

Reply:
I think we should, but don't know where. Probably, in `./testdocs` directory?

Reply:
or you can just use `npx` and avoid having a `package.json` entirely